### ISSUE TITLE
Assign maintainers in addition to requesting review

### ISSUE THREAD

**Issue by adamjstewart** [spack/spackbot#93]
Spackbot automatically requests reviews from package maintainers. However, we've also been assigning people to PRs to make it easier to visualize which PRs need attention. We should teach spackbot how to do this too. Assigning may require a higher level of permissions than requesting review, so this may need a try-except.

**Comment by alecbcs** [comment]
Yep this is a great idea. Would you suggest we assign all maintainers or only those with `merge-to-develop` permissions? 

I've thought about implementing this feature before, but with the constraint of only assigning folks who have triage permissions.

**Comment by adamjstewart** [comment]
My vote would be only people with merge-to-develop permissions
